### PR TITLE
Pops 48

### PR DESCRIPTION
**Pull request by damklis**

### CODE REVIEW

File: project-apps-dev/ProgrammersOnly/mongodb_service.tf [hunk 1]
```diff
@@ -44,7 +44,8 @@ resource "aws_ecr_repository_policy" "mongodb" {
                 "ecr:BatchDeleteImage",
                 "ecr:SetRepositoryPolicy",
                 "ecr:DeleteRepositoryPolicy"
-            ]
+            ],
+            "Resource": "*"
```
Comment (bwieckow) [inline project-apps-dev/ProgrammersOnly/mongodb_service.tf:48]: This is not needed
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository_policy

**Review by bwieckow — CHANGES_REQUESTED** [review]

File: project-apps-dev/ProgrammersOnly/mongodb_service.tf (hunk 1 above)
Comment (damklis) [reply project-apps-dev/ProgrammersOnly/mongodb_service.tf]: Done.

File: project-apps-dev/ProgrammersOnly/task_definitions/mongodb_task_definition.json [hunk 2]
```diff
@@ -12,17 +12,16 @@
...
+        "secrets": [
             {
                 "name": "MONGO_INITDB_ROOT_USERNAME",
-                "value": "root"
+                "valueFrom": "arn:aws:ssm:eu-central-1:318625651741:parameter/programmersonly/mongodb/MONGO_INITDB_ROOT_USERNAME"
             },
             {
                 "name": "MONGO_INITDB_ROOT_PASSWORD",
-                "value": "root"
+                "valueFrom": "arn:aws:ssm:eu-central-1:318625651741:parameter/programmersonly/mongodb/MONGO_INITDB_ROOT_PASSWORD"
```
Comment (bwieckow) [inline project-apps-dev/ProgrammersOnly/task_definitions/mongodb_task_definition.json:24]: Add this also to /mongodb/mongodb_task_definition.json

**Review by bwieckow — CHANGES_REQUESTED** [review]

File: project-apps-dev/ProgrammersOnly/task_definitions/mongodb_task_definition.json (hunk 2 above)
Comment (damklis) [reply project-apps-dev/ProgrammersOnly/task_definitions/mongodb_task_definition.json:24]: Added.

**Pull request merged**